packages/components/src/components/timeline/timeline.scss

```diff
@@ -20,16 +20,17 @@
     &__oval {
         width: 24px;
         height: 24px;
-        line-height: 24px;
+        line-height: 23.5px;
```

Comment:
can use `rem` instead of fractional `px`?

Reply:
👍